src/EFCore.PG/Extensions/DbFunctionsExtensions/NpgsqlDbFunctionsExtensions.cs

```diff
@@ -141,4 +141,24 @@ public static int Distance(this DbFunctions _, DateOnly a, DateOnly b)
     /// </remarks>
     public static TimeSpan Distance(this DbFunctions _, DateTime a, DateTime b)
         => throw new InvalidOperationException(CoreStrings.FunctionOnClient(nameof(Distance)));
+
+    /// <summary>
+    /// Returns the string <paramref name="value"/> as a Date
+    /// </summary>
+    /// <param name="_">The <see cref="DbFunctions"/> instance.</param>
+    /// <param name="value">The string to be converted</param>
+    /// <param name="format">The format of the input date</param>
+    /// <see href="https://www.postgresql.org/docs/current/functions-formatting.html"/>
+    public static DateOnly? ToDate(this DbFunctions _, string? value, string format)
```

Comment:
Is there any specific reason value is nullable, but not format?

Reply:
Honestly, no. Can`t really remember why I did that. I`m going to change that, as it definitely should be.